Kickstarter-iOS/Views/Controllers/MessageDialogViewController.swift

```diff
@@ -93,6 +94,17 @@ internal final class MessageDialogViewController: UIViewController {
                                animated: true,
                                completion: nil)
   }
+
+  private func postNotificationIfNeeded() {
+    guard shouldSendNotification() else { return }
```

Comment:
This is already checked in the `AppDelegateViewModel`'s `showAlert` signal right? Any reason to check it again?